src/test/java/org/apache/sling/event/impl/discovery/InitDelayingTopologyEventListenerTest.java

```diff
@@ -296,6 +298,7 @@ public void testChanged() throws Exception {
         doTestAdditionalEventsAfterInit(delegate, listener);
     }
 
+    @Ignore
```

Comment:
If this (and the test above) are skipped intentionally, could you please add a comment in the code explaining the reason, or pointer to a Jira ticket to fix them.

Reply:
I'd add that to the Ignore that, and also open a ticket.

Reply:
I'll do a final review then merge